doc/2/plugins/plugin-context/accessors/sdk/index.md

```diff
@@ -69,13 +69,72 @@ async myAwesomePipe (request) {
 
 - The created document will have the `author` metadata property set to `null`.
 
+## Realtime notifications
+
+<SinceBadge version="change-me" />
+
+The [realtime](/sdk/js/7/controllers/realtime) is entirely available with the [subscribe](/sdk/js/7/controllers/realtime/subscribe) and [unsubscribe](/sdk/js/7/controllers/realtime/unsubscribe) methods.  
+
+Realtime subscription should be made in the plugin [init](core/2/plugins/guides/manual-setup/init-function) method or in a hook on the [kuzzle:start:before](/core/2/plugins/guides/events/kuzzle-start) event.
+
+::: warning
+If you want to make subscription in hooks, pipes, or controller actions then you should use the `replicate: true` option otherwise your callback will not be executed with every notification.
+:::
+
+When you receive a notification, only one node will execute the associated callback.  
+
+::: info
+  You can use the `replicate` option if you want your callback function to be executed on each nodes.
```

Comment:
I don't get it. If plugins subscribe before kuzzle starts, then the probability that they will register the same filters is close to 1. And that means that all plugin instances should receive the notification, independantly to the value of the `replicate` option.

Reply:
I wonder if this is an architecture problem, or a semantic one: could `propagate` be a better keyword for this behavior?

Reply:
Proposal: `executionStrategy: cluster|node`

Reply:
Also hard to understand. I might want a "node" behavior in a cluster, and this new proposal can lead to the assumption that this should not be used that way.

If I think about how to document the feature to users, I think that `propagate` is simpler to understand. We could write something like this:

```
The `propagate` option allows to define if, for that subscription, notifications should be propagated to all cluster nodes, or if only the node having received the triggering event should execute it.
Values: true, false
```

wdyt?

Reply:
LGTML :+1: